uw-research-computing/transfer-data-researchdrive.md

```diff
@@ -5,31 +5,26 @@ title: Transferring Files Between CHTC and ResearchDrive
 ---
 
 UW Madison provides a shared data storage for research called [ResearchDrive](https://it.wisc.edu/services/researchdrive/). It 
```

Comment:
@rachellombardi can you review this at some point?

Reply:
@ChristinaLK Is this as a new PR? Where should I be looking to edit?

Reply:
I integrated your + my changes together. I'm happy with this document - if you are too, then we can merge it.

Reply:
I made some minor changes (e.g. removed {: .term}, clarified a few sentences). I'm happy with it if you want to merge it.